### DISCUSSION TITLE
How to (is it possible) to set "code location" attribute in Front End pipeline via aio cli?

### DISCUSSION THREAD

**Discussion by marcin-smelkowski** [adobe/aio-cli#581] [Q&A]
Hi,
as a devops I automate integration with AEM Cloud envrionment and I'd like to automat Front End pipeline reconfiguration.
Is it possible and how can I set "code location" attribute in Front End pipeline via aio cli?

BR, Marcin

**Comment by purplecabbage** [comment]
What do you mean by `code location`? The src can be specified by modifying ext.config.yaml
```
actions: actions <= where the src action code is
web: web-src <= where the src frontend code is
```

Front end is built to dist/extension-id/web-dev/ _(or web-prod/)_ and `aio app deploy` deploys it.

**Reply by purplecabbage** [reply]
You can also override the default operations with hooks.
https://developer.adobe.com/app-builder/docs/guides/app-hooks/
